cmake/ActsCompilerOptions.cmake

```diff
@@ -13,6 +13,11 @@ set(cxx_flags
     "-Wall -Wextra -Wpedantic -Wshadow -Wzero-as-null-pointer-constant -Wold-style-cast"
 )
 
+# Add assertions to standard libraries
+if(ACTS_FORCE_ASSERTIONS)
+    set(cxx_flags "${cxx_flags} -D_GLIBCXX_ASSERTIONS -D_LIBCPP_DEBUG")
```

Comment:
Is it safe to apply both flags? Should this be gated based on the stdlib version being user or is that overkill?

Reply:
Hmm doubt there are issues if the macro for GLIBCXX is present in libc++ and vise versa. I would keep it simple until this makes problems.